### PR TITLE
Run actual tests against the agent

### PR DESCRIPTION
**Pull request by sharpner**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,7 +11,13 @@ install:
 
 services:
   - mongodb
-# make mongodb a replica set
+ddons:
```
Comment (alcaeus) [inline .travis.yml]: You're missing a letter here. Also, if you're installing mongodb-org-server via addons you don't need to enable the mongodb service as those will conflict.

Comment (sharpner) [inline .travis.yml]: Copied the config from this issue: https://github.com/travis-ci/travis-ci/issues/3694 - but yeah, I removed a letter :D

Comment (alcaeus) [inline .travis.yml]: You should start copying my stuff: https://github.com/alcaeus/mongo-php-adapter/blob/master/.travis.yml :stuck_out_tongue:

Comment (sharpner) [inline .travis.yml]: Thx! Trial and error kinda sucks on travis.

**Pull request merged**